packages/integration-test/truffle-config.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable func-names */
+/* eslint-disable object-shorthand */
+require('dotenv').config();
+const { GanacheSubprovider } = require('@0x/subproviders');
+const HDWalletProvider = require('truffle-hdwallet-provider');
+const ProviderEngine = require('web3-provider-engine');
+const { toWei, toHex } = require('web3-utils');
+
+const compilerConfig = require('./compiler');
+
+const addressIndexToManage = 1;
+const numAddresses = 2;
+
+let rinkebyProvider = {};
+let mainnetProvider = {};
+let ropstenProvider = {};
+
+// You must specify a MNEMONIC and INFURA_API_KEY in your .env file
+function createProvider(network) {
+    if (process.env.CI && process.env.CIRCLE_JOB !== 'artifacts') {
+        return {};
+    }
+    if (!process.env.MNEMONIC) {
+        console.log('Please set a MNEMONIC in a .env file');
+        process.exit(1);
+    }
+    if (!process.env.INFURA_API_KEY) {
+        console.log('Please set your INFURA_API_KEY');
+        process.exit(1);
+    }
+    return () => {
+        return new HDWalletProvider(
+            process.env.MNEMONIC,
```

Comment:
This is probably why CircleCI is failing.

Reply:
Yeah, think so. Should work now we've updated